docker/docker-compose.mysql-ssl-server-ssl-connector.yml

```diff
@@ -122,6 +122,7 @@ services:
             --tls_key=/ssl.server/acra-server.key
             --tls_db_sni=mysql
             --acraconnector_transport_encryption_disable
+            --tls_auth=1
```

Comment:
why `tls_auth=1` instead 4?

Reply:
This why I don't really like numeric options for enumerations: it's not immediately obvious that 4 is an option, and what exactly it does.

When you're doing a code review of configuration changes, you either have to remember it, or be diligent enough to find the source code of the tool, find out that this number refers to [`ClientAuthType`](https://golang.org/pkg/crypto/tls/#ClientAuthType), and then you might think whether `RequireAndVerifyClientCert` is a better choice than `RequestClientCert`.

Reply:
> why tls_auth=1 instead 4?

Completely agree. While we may use `tls_auth=1` for TLS down to end application _in bounds of these simple demos_, to avoid of additional complex configuration of a test application, there are no reasons to make less secure connection between AcraServer and AcraConnector.